docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Host and deploy Blazor
+author: guardrex
+description: Discover how to host and deploy Blazor apps using ASP.NET Core, Content Delivery Networks (CDN), file servers, and GitHub Pages.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/host-and-deploy/index
+---
+# Host and deploy Blazor
+
+By [Luke Latham](https://github.com/guardrex) and [Rainer Stropek](https://www.timecockpit.com)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+## Reduction in payload size on build
```

Comment:
This feels off topic. Instead I think it makes more sense to start off with a "Publishing your app" section that describes the need to prepare your app for deployment by running `dotnet publish` like we had discussed earlier. While it's true that your IDE may handle doing the `dotnet publish` for you, I still think it's an important concept to understand so that you can figure out how to deploy to any host.

After that I think there should be a short section that summarizes the difference between hosted vs standalone deployments and states right up front that you don't need .NET on the server deploy a Blazor app. And after that would be the nice sections you have already with all the specific details on how to do hosted and standalone deployments.